dockers/docker-database/docker-database-init.sh

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env bash
 
+# For linux host namespace, in both single and multi ASIC platform use the loopback interface
+# For other namespaces, use eth0 interface which is connected to the docker0 bridge in the host.
+if [[ $NAMESPACE_ID == "" ]]
+then
+    INTFC=lo
```

Comment:
>INTFC=lo [](start = 4, length = 8)

On single ASIC, you `ip` command will get both `127.0.0.1` and a configured loopback IP address, which is reachable to the outside.

1. how to handle multiple IP addresses?
2. If you bind to 2nd one, you expose redis to public.

Reply:
In the case of single ASIC, it will fall into the if [[ $NAMESPACE_ID == "" ]] condition **alone** and we use the INTFC as **lo** and with the ip command should derive 127.0.0.1. 

Do let me know if you are referring to Loopback0 interface as in the config_db.json below ?
    "LOOPBACK_INTERFACE": {
        "Loopback0|10.1.0.32/32": {},
        "Loopback0|FC00:1::32/128": {}
    },

admin@str-s6000-acs-8 ~$ ip -4 -o addr show Loopback0 | awk '{print $4}' | cut -d'/' -f1
10.1.0.32
admin@str-s6000-acs-8 ~$ ip -4 -o addr show lo | awk '{print $4}' | cut -d'/' -f1
127.0.0.1